spec.yaml

```diff
@@ -355,42 +355,47 @@ paths:
               type: string
           schema:
             $ref: "#/definitions/Error"
-  /invite/accept:
+  /invite/forward:
     post:
-      summary: Accept an invite to a shared document
-      description: Foo
+      summary: Forward an invitation to start sharing
```

Comment:
Is there a reason for using both "invite" and "invitation" instead of just one of them?